lib/pymedphys/_experimental/streamlit/apps/film_laser_align.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2021 Jacob McAloney, Simon Biggs, Cancer Care Associates
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pymedphys._imports import streamlit as st
+
+from pymedphys._streamlit import categories
+
+CATEGORY = categories.PLANNING
+TITLE = "Film laser alignment"
+
+
+def main():
+    st.write("Hi Simon!")
```

Comment:
testing @SimonBiggs, does this work...

Reply:
Hi @JacobMcAloney 🙂

Reply:
Hi @JacobMcAloney 
Thank you for a refreshingly small/well-contained PR! (no snark, really, it's great to start small to work out the initial kinks).

It looks like the CI (the "build") is failing because the code isn't currently passing the linter.
Please make sure that you have followed the instructions for contributors (e.g. https://docs.pymedphys.com/contrib/setups/setup-linux.html)
to make sure that your pre-commit hooks are catching the things they are meant for.

If you are running on an older MacOS getting all of the dependencies installed can be a bit cumbersome (many of the brew installations have to be from source, but they do seem to build and install), and you *need* to get those dependencies installed first before installing your version of Python.  You may need to uninstall and re-install Python (or even "pyenv") if there are missing dependencies.

Then, prior to pushing your code (or creating or amending your PR), please try
`poetry run pymedphys dev lint`

Depending on your coding environment, you can configure pylint as part of "saving your file".

That should catch those issues that don't prevent your code from running properly, but do represent a potential code hygiene issue (and @SimonBiggs has set things up to be very careful about good code hygiene, one of the things I really appreciated about PyMedPhys)

When you have added a test for your new code, or if you have modified old code you will want to 
`poetry run pymedphys dev tests`

note that some of the tests appear to rely on you having pymedphys installed as a package as well as having it in source and being harnessed by pymedphys.  If you are "invoking" (calling) pymedphys as an executable within a test, things get a bit awkward if you are relying on your development version to be the one getting called.

you can make your testing more specific: (from the CHANGELOG, I had to hunt for it myself...)
pymedphys dev tests includes simplified flags. --run-only-slow can be undergone with --slow, --run-only-yarn with --cypress, --run-only-pydicom with --pydicom, and --run-only-pylinac with --pylinac. 

cypress is the test harness for streamlit based apps (such as the one you have started contributing).

I found "getting started" to take a bit of effort, and I still occasionally forget to lint or accidentally miss a test run and on a fresh clone of pymedphys, one has to re-do the pre-commit hook installation bit...  But overall, that part is pretty cookbook and makes pymedphys a lot easier to read and work with.  And the CI definitely catches things when I've "missed" something locally.

If you need help resolving certain lint or test issues, feel free to ask.
Some CI issues can be a bit obscure and not necessarily directly related to one's own code, but for the most part the detailed error messages are helpful.

Reply:
Hi @sjswerdloff 

Wow, thank you so much for the incredible information, I really appreciate it as I'm in way, way over my head here. I'd like to get better, thus trying to tackle this small project while Simon is away, but will definitely be fumbling my way through it. I will be trying to work on this during the week but it will depend on available time, so don't be surprised if nothing changes till Thursday or Friday...

In the meantime, I will try and get a handle on the matters surrounding my set-up and how I submit PRs going forward.

Again, thank you, and I appreciate the offer of support!

Reply:
Hi @sjswerdloff,

Thank you so much for holding the fort while I was gone 😄.